xmtp/src/account.rs

```diff
@@ -28,7 +32,9 @@ impl VmacAccount {
     }
 
     pub fn generate() -> Self {
-        Self::new(OlmAccount::new())
+        let mut acc = OlmAccount::new();
+        acc.generate_fallback_key();
```

Comment:
Realized I have to do this manually to ensure that a fallback key is present after generating.